src/core/server/services/stories/index.ts

```diff
@@ -66,7 +66,7 @@ export async function findOrCreate(
   scraper: ScraperQueue,
   now = new Date()
 ) {
-  let siteID = null;
+  let siteID = "";
```

Comment:
I would revert this change here, and instead below in the call to `StoryCreatedCoralEvent.publish` you use `siteID: story.siteID` instead.